code/modules/unit_tests/json_logging.dm

```diff
@@ -0,0 +1,17 @@
+/// Currently this only checks for a valid semver string
+/datum/unit_test/json_logging
```

Comment:
This test name should be more specific, I anticipate I will eventually add more tests to this file